sympy/functions/special/delta_functions.py

```diff
@@ -186,6 +186,20 @@ class Heaviside(Function):
     .. [*] Regarding to the value at 0, Mathematica defines ``H(0) = 1``,
            but Maple uses ``H(0) = undefined``
 
+    To specify the value of Heaviside at x=0, a second argument
+    can be given, and Heaviside will that value.
```

Comment:
How about:
`can be given.`~~, and Heaviside will that value~~`  Omit or pass None to recover the default unevaluated behaviour.`